.gitignore

```diff
@@ -194,3 +194,6 @@ pnpm-debug.log*
 *.njsproj
 *.sln
 *.sw?
+
+package-lock.json
+yarn.lock
```

Comment:
https://github.com/yarnpkg/yarn/issues/1583

dont ignore yarn.lock

Reply:
while we use yarn as the package manager, please ignore npm's lock file

Reply:
> while we use yarn as the package manager, please ignore npm's lock file

I do recommend keeping **both** files. So if a contributor uses npm instead of yarn they can still have all the dependencies sorted out.